<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test</class>
 <widget class="QWidget" name="Test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>66</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Label</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="IntEditor" name="intEditor" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DoubleEditor" name="double_editor" native="true"/>
   </item>
   <item>
    <widget class="TwoIntEditor" name="two_int_editor" native="true"/>
   </item>
   <item>
    <widget class="TwoDoubleEditor" name="two_double_editor" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IntEditor</class>
   <extends>QWidget</extends>
   <header>./widgets/editors/number_editor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DoubleEditor</class>
   <extends>QWidget</extends>
   <header>./widgets/editors/number_editor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TwoIntEditor</class>
   <extends>QWidget</extends>
   <header>./widgets/editors/number_editor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TwoDoubleEditor</class>
   <extends>QWidget</extends>
   <header>./widgets/editors/number_editor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
